docs/tutorials/Forecast Uber Stock Prices using MindsDB.mdx

```diff
@@ -0,0 +1,89 @@
+# Introduction
+When it comes to AutoML, one of the most remarkable open-source products is MindsDB. MindsDB enhances the existing databases and modifies their tables to function as AI Tables that can train robust Predictor models using the available data, without the need to learn how to code such models in Python, handle their dependencies, or maintain them at a later stage. With MindsDB, you can use simple SQL-like statements and obtain accurate predictions with just a few clicks.
+
+In this tutorial, we will learn how to forecast the stock prices of Uber using MindsDB. The required dataset for this tutorial can be downloaded from here.
```

Comment:
Looks like you forgot to add a dataset download link here.

Reply:
dataset added